### DISCUSSION TITLE
Corporations and Other Types of Business Entities

### DISCUSSION THREAD

**Discussion by WisTex** [schemaorg/schemaorg#2945] [General]
As I was writing up the markup for a company, I came upon an unexpected dilemma. Do I tag a limited liability company as a "Corporation" or an "Organization?"

The description on https://schema.org/Corporation wasn't that helpful.

Technically and legally, they are not corporations. But they basically serve the same purpose as a corporation, and a layperson would lump LLCs and corporations into the same bucket.

So, as far as schema.org is concerned, is a limited liability company, or a limited partnership, or other similar structure a "corporation" or an "organization" or something else? How are you defining the word "Corporation" in this context?

And should there be an entity type field where you can specify the type of organization?

**Comment by RichardWallis** [comment]
The only markup difference between an Organization and its Corporation subtype is that Corporation has the additional property [tickerSymbol](https://schema.org/tickerSymbol).

**Comment by thadguidry** [comment]
Hi @WisTex 
Good questions.  Short answer is we never went down the rabbit hole too much within the domains of Law.  Henceforth...**incorporation**

The important context is that of **incorporation** and not necessarily Corporation.
Legally tying together several entities in order to form 1 virtual entity for tax or legal reasons, like separating assets and income from its owners and investors. [[ref.](https://www.investopedia.com/terms/i/incorporate.asp)]
Where the tying together is formalized in documents and stored with a  registration authority (county/state/province/country, etc.)

The 2 most important properties that you have available now are [legalName](https://schema.org/legalName) and [leiCode](https://schema.org/leiCode).
LEI is especially important because it's the first and only open, non-proprietary legal entity identification system designed as a public good.

We could add an additional property called **entityLegalForm** (and probably registeredAt and jurisdicitonOfFormation) just as LEI has, but we did not (yet).
Example records:
https://search.gleif.org/#/record/549300VRR0S0ANY77J68
https://search.gleif.org/#/record/549300KIK0XCORRK5393
https://search.gleif.org/#/record/9884002AESDO4YW87G32
https://search.gleif.org/#/record/549300FZFNEICTT28I40

Additional info:  https://www.gleif.org/en/about-lei/semantic-representation-of-the-lei/lei-model-in-rdf-resource-description-framework#

**Reply by WisTex** [reply]
Thanks for clearing that up in regards to "corporation" meaning an incorporated or registered entity.

What's interesting is that LEI is basically the new DUNS. Both LEI and DUNS have the same exact purpose, to identify global entities and track who owns who. What's weird is the proprietary one (DUNS) is free, and the non-proprietary one (LEI) is not. Go figure. And LEI uses proprietary companies to register LEI numbers, so it's basically exactly like a DUNS number. They just have a different revenue model.

**Comment by thadguidry** [comment]
@RichardWallis We probably should improve the description of **https://schema.org/Corporation** and at least a small mention of the legal process termed "incorporation".

**Reply by WisTex** [reply]
Giving some examples on https://schema.org/Corporation might be useful too.

**Comment by RichardWallis** [comment]
Looking to someone(s) with more knowledge in this are than me for suggestions for a more informative description and possibly some examples too.

**Reply by thadguidry** [reply]
@RichardWallis I think the Wikipedia article gives quite a bit of background:  https://en.wikipedia.org/wiki/Incorporation_(business)

One plausible description:

**Corporation -**
**a legally defined entity that has been created within a particular jurisdiction through a process of incorporation or company formation (Articles of Incorporation, charter, certificate of incorporation, letters patent, etc.)  Ref: https://en.wikipedia.org/wiki/Incorporation_(business)**

**Reply by WisTex** [reply]
Although different terms are used, the general hierarchy worldwide is as follows:

1. Corporation (Corp, Inc., Ltd., AG, etc.)
2. Limited Liability Company (LLC, GmbH, etc.)
3. Limited Partnerships (LP)
4. Partnerships
5. Sole Proprietorships / Sole Traders

The first two (corporations and LLCs) are legally different types of entities, but both are required to be registered with the state and both create a distinct legal entity that is separate from its owners.

Four and five (partnerships and sole traders) are not entities separate from their owners and the owners assume 100% liability. In most jurisdictions, partnerships and sole proprietorships are required to be registered to use a name other than their owner(s), such as a DBA, assumed name, or trade name, but they are not considered separate entities from their owner(s).

Three is basically a partnership that has been granted limited liability. It's usually not an entity separate from its owners either, although it can make contracts in the partnership's name.

So, if we are using an expanded definition, perhaps we should go with something like this:

* "an entity that is legally separate from its owners, such as a corporation or limited liability company."

That way partnerships and sole proprietorships, even if registered with the state, are not included.

**Comment by thadguidry** [comment]
@WisTex
We'd likely want to avoid deep hierarchy and also verbiage of any "liability" context in Schema.org descriptions.  As well as not offering any suggested (enumerated) legal forms. The laws are too fluid in many countries of what is/is not an "incorporated entity" or what defines "legally separate".
What is better is to offer the 2 or 3 components that are missing.  As I stated a few comments earlier that a new property **entityLegalForm** could be introduced to capture more detail of the legal form of an entity as defined in a Jurisdiction.  This would also likely need to be a property with entirely new Type in order to offer the right multi-valued context for any given [Schema.org/Corporation](https://schema.org/Corporation).  Something like

**LegalEntity** :
- **legalName**
- **entityLegalForm**
- **jurisdiction**
- **jurisdictionRegistrationNumber**
- **leiCode**

which is identical to what the EU, LEI, and USA ( through [the Securities and Exchange Act of 1934](https://en.wikipedia.org/wiki/United_States_corporate_law)) basically frame up.

----

@RichardWallis 
If we had to simplify things right now, the current understanding of [Schema.org/Corporation](https://schema.org/Corporation) is:
> an incorporated legal entity in a jurisdiction

or use my proposed longer description:
> a legally defined entity that has been created within a particular jurisdiction through a process of incorporation or company formation (Articles of Incorporation, charter, certificate of incorporation, letters patent, etc.) Ref: https://en.wikipedia.org/wiki/Incorporation_(business)

We do not currently have the type **LegalEntity** but it could be introduced and [Schema.org/Corporation](https://schema.org/Corporation) would be a subtype of **LegalEntity**.

However, let's address the direct need first, that of improving the Corporation description to be a touch more accurate.
Then we can additionally add a new type and a few properties to help qualify a specific **LegalEntity** just as [LEI](https://www.gleif.org/en/) and [OpenCorporates](https://opencorporates.com/), [DNB.com](https://www.dnb.com/) and others do.

We currently do not have a way to say anything about a **LegalEntity** other than [legalName](https://schema.org/legalName) and [leiCode](https://schema.org/leiCode)  (it must also include legalForm and jurisdiction, otherwise the right context will be lost and ambiguity introduced)

### OpenCorporates examples:

Schema.org/LegalEntity
- name: [Microsoft Corporation](https://opencorporates.com/companies/us_wa/600413485)
- legalForm: WA Profit Corporation
- jurisdiction: Washington (US)
- jurisdictionRegistrationNumber: 600413485
- registeredWith: https://www.sos.wa.gov

Schema.org/LegalEntity
- name: [Microsoft Corporation](https://opencorporates.com/companies/us_de/2102063)
- legalForm: Corporation
- jurisdiction: Delware (US)
- jurisdictionRegistrationNumber: 2102063
- registeredWith: https://corp.delaware.gov/

**Reply by WisTex** [reply]
I agree that we need to create a new type called "LegalEntity" for situations like these, because I think the goal is to identify "who" regardless of whether it is a corporation or not.

* LegalEntity - Any organization registered with the government and that has a legal existence.
* Corporation - A specific type of registered organization / legal entity.
* Organization - Any type of organization, including a division or department within a corporation, partnership, non-profit, or other registered organization.

DUNS and LEI should both be included. In most countries, active registered businesses get a DUNS number without even asking for it and it's free. LEI is mostly for entities that deal with financial markets. Small businesses almost never need a LEI, but small businesses do get assigned a DUNS number.
